src/components/TodoInfo/TodoInfo.scss

```diff
@@ -0,0 +1,18 @@
+.TodoInfo {
+  width: max-content;
+  padding: 8px;
+  margin: 12px 0;
+  border: 1px solid #000;
+  border-radius: 8px;
+  background-color: antiquewhite;
+
+  &__title {
+    margin: 4px 0;
+    font-size: inherit;
+    color: #f00;
+  }
+
+  &--completed &__title {
+    color: #080;
```

Comment:
It is a good practice to make all colors variable

Reply:
it was by default, I did not change it

Reply:
after the fork of the task, it is already your project, so you are also responsible for all the code and its development

Reply:
understood, just before that it was written "you do not need to change scss or html"
but i got u

Reply:
accepted